### PR TITLE
updated the partner doc with latest changes usage of OAuth

### PR DESCRIPTION
**Pull request by varunutreja**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/partner/index.md [hunk 1]
```diff
@@ -414,103 +396,34 @@ A typical event will have the following information.
...
...access_token": "eyJhbGciOiJSUzI1NiIsIng1dSI6Imltc19uYTEtc3RnMS1rZXktYXQtMS5jZXIiLCJraWQiOiJpbXNfbmExLXN0ZzEta2V5LWF0LTEiLCJpdHQiOiJhdCJ9.eyJpZCI6IjE2OTU4OTQ3NTYwOTdfYWYyMWQ2OTktNzkwNy00NjNmLThhOTktODk5Njc3OTUxNTE3X2V3MSIsIm9yZyI6IjM1QTk0M0MwNjJEQTY0MTEwQTQ5NDIzN0BBZG9iZU9yZyIsInR5cGUiOiJhY2Nlc3NfdG9rZW4iLCJjbGllbnRfaWQiOiIyYmIyZmViMGZkZWM0ZmRlODk1MTY0NGJhNzcwZTMyOSIsInVzZXJfaWQiOiI3OTlCMUE5MjY1MDJFNEM1MEE0OTQwMDlAdGVjaGFjY3QuYWRvYmUuY29tIiwiYXMiOiJpbXMtbmExLXN0ZzEiLCJhYV9pZCI6Ijc5OUIxQTkyNjUwMkU0QzUwQTQ5NDAwOUB0ZWNoYWNjdC5hZG9iZS5jb20iLCJjdHAiOjMsIm1vaSI6IjcwNTg2OWI3IiwiZXhwaXJlc19pbiI6Ijg2NDAwMDAwIiwiY3JlYXRlZF9hdCI6IjE2OTU4OTQ3NTYwOTciLCJzY29wZSI6ImFkb2JlaW9fYXBpLG9wZW5pZCxyZWFkX2NsaWVudF9zZWNyZXQsQWRvYmVJRCxhZGRpdGlvbmFsX2luZm8ucm9sZXMsbWFuYWdlX2NsaWVudF9zZWNyZXRzLHJlYWRfb3JnYW5pemF0aW9ucyxhZGRpdGlvbmFsX2luZm8ucHJvamVjdGVkUHJvZHVjdENvbnRleHQifQ.g3eBFADKN6q9gxMiiAL53MxM7FQGDO3LSahtpcp2E7irEJ6xWjrhc28rFEFoZZc3rRBDWiUqGj2faVopUZmNZ0HbKl-QcXfM9yYRzsBm9y8Zw91RbvieNU1MqxkYbCS-TzcFBAgGGPcdyyWylT_2WgAIUiXI9K5CyR2tRaScqwFhAS8S371AA7Hq3QMyJUGwZwqxYMbfKy2jh1ZNYyCLkRHXMwl2FUNgjk2pQiOnetbSsI0muxwu_cOoE8i6A9TA8CY4vZ22zSCCaQXXeKG2jGB1qGEz1CaRCEutYPcY-Lxx1YwvN73L7nrLR8_yZ5_2MKK_fm9VjhmViQHgsQ",
```
Comment (vikchau) [inline src/pages/partner/index.md:418]: I hope this is altered enough not be an actual token.

Comment (varunutreja) [reply src/pages/partner/index.md:418]: Yes, I'd removed few characters in between.

File: src/pages/partner/index.md [hunk 2]
```diff
@@ -41,173 +44,149 @@ The following workflow diagram describes the steps in the partner registration p
...
-2. **Get an IMS token:** The Globalization Content Service team procures an IMS bearer token for your firm. Use this token to get authenticated through the Adobe I/O console. 
-3. **Log on the Adobe I/O Console**: Use the IMS token to log in Adobe I/O console. Once verified, you can access the products and services based on the permissions granted to you. 
-4. **Subscribe to APIs**: You can subscribe to the Adobe I/O Globalization Content Service APIs to manage projects, tasks, and assets. 
-5. **Subscribe to events**: Using Adobe I/O journaling APIs, subscribe to events and get near real-time communication. Even if you have thousands of tasks needing localization every week, the event journal will help you get info as soon as tasks are being triggered at the product end. Based on task information received in events, retrieve assets, localize them, preview quality, and send final translated assets back to the product through the Adobe I/O Globalization Content Service APIs. 
+1. **Get an IMS token:** The Globalization Content Service team procures an IMS bearer token for your firm. Use this token to get authenticated through the Adobe I/O console.
```
Comment (vikchau) [inline src/pages/partner/index.md]: @varunutreja Is this not self serve. Anyone can come and create an account to use with the dev console. Do we have to procure the token?

File: src/pages/partner/index.md [hunk 3]
```diff
@@ -41,173 +44,149 @@ The following workflow diagram describes the steps in the partner registration p
...
+2. **Log on the Adobe I/O Console**: Use the IMS token to log in Adobe I/O console. Once verified, you can access the products and services based on the permissions granted to you.
```
Comment (vikchau) [inline src/pages/partner/index.md]: How does one use a token to log in to the console. From a UI, the only options I see are through username and password.

File: src/pages/partner/index.md [hunk 4]
```diff
@@ -41,173 +44,149 @@ The following workflow diagram describes the steps in the partner registration p
...
+7. Generate OAuth Server-to-Server access token by clicking on to the CREDENTIALS section [We have redacted it.]
 
-9. Select **Save configured API** to save Globalization Content Service APIs enablement in the project.
+   ![Create A OAuth Server-to-Server](resources/ss_Create_OAuth.png)
 
-    ![JWT Credentials](resources/ss_Service_Account.png)
+The new OAuth Server-to-Server credentials rely on the OAuth 2.0 client credentials grant. Therefore, you can use industry-standard OAuth 2.0 libraries to implement access token generation in your application.
 
-10. Generate a JWT token by using the private key. Click Service Account (JWT). 
+Unlike the Service Account (JWT) credential, the new credential does not use a public certificate and private key pair to generate access tokens. As an application developer, you do not have to periodically rotate the public certificates and private key pairs when they expire. Also, the credential setup process is greatly simplified, and you do not have to download and save the private key on your machine.
```
Comment (vikchau) [inline src/pages/partner/index.md]: References to JWT may not be needed.

File: src/pages/partner/index.md [hunk 5]
```diff
@@ -41,173 +44,149 @@ The following workflow diagram describes the steps in the partner registration p
...
 
-    Create a new service account (JWT) credential. Here, you create a JSON Web Token (JWT) that includes your credentials. You’ll begin your sessions by exchanging the JWT for an access token at Adobe I/O console. Once access is granted, you start using Globalization Content Service events. Since you created a public key while registering to Globalization Content Service APIs, you can use it to create JWT.
+6. The **Configure event registration** screen shows four events that you can subscribe.
 
-8. Select **Upload your public key.**
-
-    ![Create JWT event](resources/ss_Create_JWT_event.png)
+   1. **LEVERAGE_TM**: Used to extract text for localization out of an Adobe asset. Also, access the translation memory (TM) to enable the optimum re-use of a previously translated content.
+   2. **PROJECT_UPDATE**: Used to update the project configuration information. The configuration data can include info on locales, content provider system and so on.
+   3. **TRANSATE**: Used to trigger an event when a task is sent for localization.
```
Comment (vikchau) [inline src/pages/partner/index.md]: Typo - TRANSLATE

File: src/pages/partner/index.md [hunk 6]
```diff
@@ -41,173 +44,149 @@ The following workflow diagram describes the steps in the partner registration p
...
+   4. **UPDATE_TM**: Used to update the translation memory. During the asset completion of manual translation, translated strings are saved in TM so that they can be used later if similar strings appear in a future task. It helps to reduce the cost of manual translation.
 
-9. Drag and drop the public key (***certificate\_pub.crt***) and select **Next**.
+   ![Configure events](resources/ss_Configure_event_registration.png)
 
-    ![Upload Public Key](resources/ss_Upload_Public_Key.png)
+7. Select the events you want to subscribe and press **Next**.
```
Comment (vikchau) [inline src/pages/partner/index.md]: Select the events you want to subscribe *to*

**Review by vikchau — CHANGES_REQUESTED** [review]
Some minor stuff. Otherwise looks good. I ran it and layout also looks fine. Thanks

File: src/pages/partner/index.md (hunk 2 above)
Comment (varunutreja) [reply src/pages/partner/index.md]: No we don't need token. Removed this line.

File: src/pages/partner/index.md (hunk 3 above)
Comment (varunutreja) [reply src/pages/partner/index.md]: Right, updated this point.

File: src/pages/partner/index.md (hunk 4 above)
Comment (varunutreja) [reply src/pages/partner/index.md]: JWT reference removed

File: src/pages/partner/index.md (hunk 5 above)
Comment (varunutreja) [reply src/pages/partner/index.md]: done

File: src/pages/partner/index.md (hunk 6 above)
Comment (varunutreja) [reply src/pages/partner/index.md]: done

File: src/pages/partner/index.md [hunk 7]
```diff
@@ -99,7 +98,7 @@ Here’s your guide to onboarding on the Adobe I/O console and accessing the Glo
 
 The new OAuth Server-to-Server credentials rely on the OAuth 2.0 client credentials grant. Therefore, you can use industry-standard OAuth 2.0 libraries to implement access token generation in your application.
```
Comment (vikchau) [inline src/pages/partner/index.md]: We should just say:

OAuth Server-to-Server credentials rely on the OAuth 2.0 client credentials grant. Therefore, you can use industry-standard OAuth 2.0 libraries to implement access token generation in your application.

While OAuth Server-to-Server credentials do not use expiring certificates, they still allow client secret rotation through the UI and API.

That's it.

**Review by vikchau — CHANGES_REQUESTED** [review]
Looks good. Just one change which I commented on. Thanks.

File: src/pages/partner/index.md (hunk 7 above)
Comment (varunutreja) [reply src/pages/partner/index.md]: done

**Review by vikchau — APPROVED** [review]

**Pull request merged**